components/cd-disclosure/cd-disclosure.js

```diff
@@ -0,0 +1,81 @@
+(function (Drupal) {
+  'use strict';
+
+  /**
+   * @see https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/
+   */
+  Drupal.behaviors.cdDisclosure = {
+    idPrefix: 'cd-disclosure--',
+
+    attach: function (context, settings) {
+      // Initialize toggable dropdown.
+      this.initializeDisclosures();
+    },
+
+    /**
+     * Set up all Disclosure widgets found in the DOM.
+     *
+     * For our purposes, the `disclosure` element is the wrapped content itself
+     * and the button to toggle it is generated automatically by this script.
+     */
+    initializeDisclosures: function () {
+      let disclosures = document.querySelectorAll('.cd-disclosure');
```

Comment:
We need to set an attribute or class like `cd-closure--processed` and change the selector to exclude processed elements otherwise they may be processed again when the script is loaded again via ajax for example.

Reply:
I was planning on re-using the attribute inside Dropdown `data-cd-processed` for consistency.